schemas/kmp/README.md

```diff
@@ -1,8 +1,15 @@
 # kmp.schema.json
 
+* kmp.json file format, metadata included in Keyman .kmp package files
+
 Documentation at https://help.keyman.com/developer/current-version/reference/file-types/metadata
 
-# version history
+# kmp.schema.json version history
+
+## 2023-10-19 2.0
+* Add relatedPackages, options.licenseFile, options.welcomeFile,
+  keyboard.examples, keyboard.webOskFonts, keyboard.webDisplayFonts,
+  info.description (all of these formerly were stored in .keyboard_info)
```

Comment:
Does this also include:
keyboardExample (id, keys, text, note)
relatedPackage (id, relationship)?

Reply:
Yes, see keyboard.examples, relatedPackages listed above, which reference those. We don't need to list every subelement here, just highest level element that changes.